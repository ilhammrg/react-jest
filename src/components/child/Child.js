import React from 'react';

const Child = () => {
  return (
    <div>
      <h2 className='child'>This is Child component</h2>
    </div>
  );
}

export default Child;