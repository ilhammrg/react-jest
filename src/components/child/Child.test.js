import React from 'react';
import { shallow } from 'enzyme';
import Child from './Child';

describe('Child component test', () => {
  test('should render Child component', () => {
    expect(shallow(<Child />)).toMatchSnapshot();
  });
});