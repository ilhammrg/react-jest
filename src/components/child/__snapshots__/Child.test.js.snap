// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Child component test should render Child component 1`] = `
<div>
  <h2
    className="child"
  >
    This is Child component
  </h2>
</div>
`;
